Dockerfiles/agent/windows/amd64/Dockerfile

```diff
@@ -1,16 +1,24 @@
 ARG BASE_IMAGE=mcr.microsoft.com/powershell:lts-windowsservercore-1809
-FROM ${BASE_IMAGE}
-
 ARG WITH_JMX="false"
 ARG VARIANT="unknown"
 
+FROM mcr.microsoft.com/powershell:lts-windowsservercore-${VARIANT} as unzipper
+USER ContainerAdministrator
+SHELL ["pwsh", "-Command", "$ErrorActionPreference = 'Stop';"]
+COPY datadog-agent-latest.amd64.zip uncompress-zip.ps1 ./
+RUN . ./uncompress-zip.ps1

```

Comment:
I can be good to:
* move previous line (ARG BASE_IMAGE, ARG WITH_JMX ...) after this block 
* wrap this part in comments to clearly identify the multi-stage build